commands/kubernetes.go

```diff
@@ -238,17 +238,17 @@ func kubernetesNodePools() *Command {
 	AddStringFlag(cmdKubeNodePoolUpdate, doctl.ArgTag, "", "",
 		"tags to apply to the node pool, repeat to add multiple tags at once")
 
-	cmdKubeNodePoolRecycle := CmdBuilder(cmd, RunKubernetesNodePoolRecycle,
-		"recycle <cluster-id|cluster-name> <pool-id|pool-name>", "recycle nodes in a node pool", Writer, aliasOpt("r"))
-	AddStringFlag(cmdKubeNodePoolRecycle, doctl.ArgNodePoolNodeIDs, "", "",
-		"ID or name of the nodes in the node pool to recycle")
-
 	cmdKubeNodePoolDelete := CmdBuilder(cmd, RunKubernetesNodePoolDelete,
 		"delete <cluster-id|cluster-name> <pool-id|pool-name>",
 		"delete node pool from a cluster", Writer, aliasOpt("d", "rm"))
 	AddBoolFlag(cmdKubeNodePoolDelete, doctl.ArgForce, doctl.ArgShortForce,
 		false, "force node pool delete")
 
+	cmdKubeNodeDelete := CmdBuilder(cmd, RunKubernetesNodeDelete, "delete-node <cluster-id|cluster-name> <pool-id|pool-name> <node-id>", "delete node in a node pool", Writer)
+	AddBoolFlag(cmdKubeNodeDelete, doctl.ArgForce, doctl.ArgShortForce, false, "force node delete")
+	AddBoolFlag(cmdKubeNodeDelete, "skip-drain", "", false, "skip draining the node before deletion")
+	AddBoolFlag(cmdKubeNodeDelete, "replace", "", false, "replace the node with a new one")
```

Comment:
I'm on the fence about whether this should be a flag or a separate command. If a user wants to replace a node, it doesn't feel entirely intuitive to look under the `delete` command, especially since in the control panel we offer both `Delete` and `Recycle` options.

I'm all for getting rid of the `recycle` terminology since we know it's confusing, but I wonder whether we should have both a `delete-node` command and a `replace-node` command.

Not a blocker for me but interested in other opinions. @hilary @andrewsomething any thoughts?

Reply:
Removing the `recycle` command is a breaking change. You can deprecate it, if you wish, and we could remove it in v2.0, but removing it in a single release without messaging and semantic versioning, etc., isn't conducive to a great developer experience for our community :)

Reply:
As for the design question, when a semantic question like this feels ambiguous, I'm a fan of expressiveness. I would offer both. Some folks will look for a flag, others will look for a separate command. So alias the separate command to the flag.

Reply:
Ok, checked in and the recycle command is seeing usage so we do need to go through the deprecation process. Not a lot of use, but some.